modern/src/reports/EventReportPage.js

```diff
@@ -1,111 +1,88 @@
 import React, { useState } from 'react';
-import MainToolbar from '../MainToolbar';
-import { Grid, TableContainer, Table, TableRow, TableCell, TableHead, TableBody, Paper, makeStyles } from '@material-ui/core';
+import { TableContainer, Table, TableRow, TableCell, TableHead, TableBody, Paper } from '@material-ui/core';
 import { FormControl, InputLabel, Select, MenuItem } from '@material-ui/core';
 import t from '../common/localization';
 import { formatPosition } from '../common/formatter';
 import ReportFilter from './ReportFilter';
-
-const useStyles = makeStyles(theme => ({
-  root: {
-    height: '100%',
-    display: 'flex',
-    flexDirection: 'column',
-  },
-  content: {
-    flex: 1,
-    overflow: 'auto',
-    padding: theme.spacing(2),
-  },
-  form: {
-    padding: theme.spacing(1, 2, 2),
-  },
-}));
+import ReportView from './ReportView';
 
 const EventReportPage = () => {
-  const classes = useStyles();
-  const [data, setData] = useState([]);
-  const [eventType, setEventType] = useState(['allEvents']);
 
-  const handleSubmit = async (deviceId, from, to) => {
-    const query = new URLSearchParams({
-      deviceId,
-      from: from.toISOString(),
-      to: to.toISOString(),
-    });
-    eventType.map(t=>query.append('type',t));
-    const response = await fetch(`/api/reports/events?${query.toString()}`, { headers: { Accept: 'application/json' } })
+  const ReportFilterForm = ({ onResult }) => {
```

Comment:
Same for this.

Reply:
Changes done. Please review them. Also changed one file name from "ReportView" to "ReportLayoutPage".